x-pack/legacy/plugins/siem/public/components/page/network/network_http_table/columns.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import numeral from '@elastic/numeral';
+import { NetworkHttpFields, NetworkHttpItem } from '../../../../graphql/types';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { getEmptyTagValue } from '../../../empty_value';
+import { IPDetailsLink } from '../../../links';
+import { Columns } from '../../../paginated_table';
+
+import * as i18n from './translations';
+import { getRowItemDraggable, getRowItemDraggables } from '../../../tables/helpers';
+export type NetworkHttpColumns = [
+  Columns<NetworkHttpItem['methods']>,
+  Columns<NetworkHttpItem['domains']>,
+  Columns<NetworkHttpItem['path']>,
+  Columns<NetworkHttpItem['statuses']>,
+  Columns<NetworkHttpItem['lastHost']>,
+  Columns<NetworkHttpItem['lastSourceIp']>,
+  Columns<NetworkHttpItem['requestCount']>
+];
+
+export const getNetworkHttpColumns = (tableId: string): NetworkHttpColumns => [
+  {
+    field: `node.${NetworkHttpFields.methods}`,
+    name: i18n.METHOD,
+    render: methods =>
+      Array.isArray(methods) && methods.length > 0
+        ? getRowItemDraggables({
+            attrName: 'http.request.method',
+            displayCount: 3,
+            idPrefix: escapeDataProviderId(`${tableId}-table-methods`),
```

Comment:
Looks like the columns can end up with dupe identifiers, which in addition to the below, will result in a slew of console errors saying `cannot unregister Droppable`:

![http_dupe_dnd](https://user-images.githubusercontent.com/2946766/68059851-ac3cd180-fcc3-11e9-98b9-d9235f930ae3.gif)

Should just be able to update `getRowItemDraggables` to include the `rowIndex` in the ID, as _weirdly enough_, it's not currently added to the id:

https://github.com/elastic/kibana/blob/58c1f879f7763fb46af6f2644040ca1f39b2976f/x-pack/legacy/plugins/siem/public/components/tables/helpers.tsx#L91-L92

Reply:
this doesnt clear it up as the index for all those is `0` anyways... working on a solution